### ISSUE TITLE
Translate app in English. 

### ISSUE THREAD

**Issue by raullgiordano** [yaklang/yakit#126]
Sir,
It looks like the app is awesome from screenshots. But after executing `./yakit-***` I can not understand any thing because the UI is in Chinese. It would be nice if there is an option to change the App UI language into English.
Thanks.

**Comment by VillanCh** [comment]
Thanks for your STAR. We are happy to hear the first international voice for this repos.

Indeed, the dev team is too small to afford the translation of entire yaklang.io. But i18n(at least English supporting) is on our task board all the time.

The current project, yakit desktop(Electron Application) is the most should be translated to English at least. First of ALL, we will provide the English README.md, then the Yakit application 's capabilities will support i18n tech page by page. Still now, our team are working on Dev & Test & PoC project almostly always, for this, the i18n / advertising processing is slow ... Apologizing for the inconvenience.

Besides, the embedded nuclei (projectdiscovery.io) is working on Yakit, which is a great chance to link other security teams.

**Comment by MedoX71T** [comment]
Yes it will be good if translated in English its powerful project i respect the developers..

**Comment by wifiuk** [comment]
i'd be interested in trying this app out if it was in English so I can read it

**Comment by connorrutt** [comment]
@VillanCh 
你好、
我是一家中国公司的英语流利的红队成员。我在美国，而这个项目似乎真的是我感兴趣的东西。如果得到批准，我可以做英文翻译。

**Comment by Vernon818** [comment]
@connorrutt 麻烦师傅加一下我微信Juzi19880818

**Comment by connorrutt** [comment]
@Vernon818 我加了你的好友 coreyin

**Comment by ip-rw** [comment]
https://github.com/yaklang/yakit/issues/1255#issuecomment-2133486434

Is an option for people who just want to understand whats going on and don't speak Chinese
